AdminWebsite/AdminWebsite.UnitTests/Mappers/NotificationMappers/MapToDemoOrTestNotificationTests.cs

```diff
@@ -187,7 +187,7 @@ public void Should_map_ejud_joh_demo_or_test_notification()
             var participant = new ParticipantResponse
             {
                 Id = Guid.NewGuid(),
-                Username = "testusername@hmcts.net",
+                Username = "contact@judiciary.hmcts.net",
                 CaseRoleName = "caserolename",
                 ContactEmail = "contact@judiciary.hmcts.net",
```

Comment:
Consider using a different value for ContactEmail to Username (Multiple instances), ideally we want each value to be unique unless there is a need for there to be duplicates.

Reply:
Sure.